src/commands/_blocks.live.ts

```diff
@@ -452,5 +600,7 @@ function renderCompactLiveDisplay(
 
 	// Footer
 	terminal.write(`${'─'.repeat(width)}\n`);
-	terminal.write(pc.gray(`Refresh: ${config.refreshInterval / 1000}s | Ctrl+C: stop\n`));
+	terminal.write(
+		pc.gray(`Refresh: ${config.refreshInterval / 1000}s | Ctrl+C: stop\n`),
+	);
```

Comment:
I'm curious. Is those style fix is happened by eslint or by you?

Reply:
I couldn't get `eslint --fix` to work. it just gave me errors, so I ran `prettier` with tab formatting. If you know how I can get eslint to fix this, i'll update.

Reply:
did you rollback the format issue?

Reply:
thank you. sorry about that. my editor is configured to automatically run prettier, so it wrecked the whole thing after I made the code changes.

Reply:
no worries, otherwise lgtm